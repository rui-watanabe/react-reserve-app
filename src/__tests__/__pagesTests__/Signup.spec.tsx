import React from 'react';
import renderer from 'react-test-renderer';
import Signup from '../../pages/signup';

test('Signup', () => {
  const component = renderer.create(<Signup />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
