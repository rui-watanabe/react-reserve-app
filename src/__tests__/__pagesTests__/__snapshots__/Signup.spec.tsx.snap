// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup 1`] = `
Array [
  <div
    className="ui teal icon attached message"
  >
    <i
      aria-hidden="true"
      className="settings icon"
      onClick={[Function]}
    />
    <div
      className="content"
    >
      <div
        className="header"
      >
        Get Started!
      </div>
      <p>
        Create a new account
      </p>
    </div>
  </div>,
  <form
    className="ui form"
    onSubmit={[Function]}
  >
    <div
      className="ui error message"
    >
      <div
        className="content"
      >
        <div
          className="header"
        >
          Oops!
        </div>
        <p>
          
        </p>
      </div>
    </div>
    <div
      className="ui segment"
    >
      <div
        className="field"
      >
        <label>
          Name
        </label>
        <div
          className="ui fluid left icon input"
        >
          <input
            name="name"
            onChange={[Function]}
            placeholder="Name"
            type="text"
            value="test"
          />
          <i
            aria-hidden="true"
            className="user icon"
            onClick={[Function]}
          />
        </div>
      </div>
      <div
        className="field"
      >
        <label>
          Email
        </label>
        <div
          className="ui fluid left icon input"
        >
          <input
            name="email"
            onChange={[Function]}
            placeholder="Email"
            type="email"
            value="test@gmail.com"
          />
          <i
            aria-hidden="true"
            className="envelope icon"
            onClick={[Function]}
          />
        </div>
      </div>
      <div
        className="field"
      >
        <label>
          Password
        </label>
        <div
          className="ui fluid left icon input"
        >
          <input
            name="password"
            onChange={[Function]}
            placeholder="Password"
            type="password"
            value="33333333"
          />
          <i
            aria-hidden="true"
            className="lock icon"
            onClick={[Function]}
          />
        </div>
      </div>
      <button
        className="ui orange disabled button"
        disabled={true}
        onClick={[Function]}
        tabIndex={-1}
        type="submit"
      >
        <i
          aria-hidden="true"
          className="signup icon"
          onClick={[Function]}
        />
        Signup
      </button>
    </div>
  </form>,
  <div
    className="ui warning bottom attached message"
  >
    <i
      aria-hidden="true"
      className="help icon"
      onClick={[Function]}
    />
    Exiting user?
     
    <a
      href="/login"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Log in here
    </a>
    
    instead.
  </div>,
]
`;

exports[`Signuped 1`] = `
Array [
  <div
    className="ui teal icon attached message"
  >
    <i
      aria-hidden="true"
      className="settings icon"
      onClick={[Function]}
    />
    <div
      className="content"
    >
      <div
        className="header"
      >
        Get Started!
      </div>
      <p>
        Create a new account
      </p>
    </div>
  </div>,
  <form
    className="ui form"
    onSubmit={[Function]}
  >
    <div
      className="ui error message"
    >
      <div
        className="content"
      >
        <div
          className="header"
        >
          Oops!
        </div>
        <p>
          
        </p>
      </div>
    </div>
    <div
      className="ui segment"
    >
      <div
        className="field"
      >
        <label>
          Name
        </label>
        <div
          className="ui fluid left icon input"
        >
          <input
            name="name"
            onChange={[Function]}
            placeholder="Name"
            type="text"
            value=""
          />
          <i
            aria-hidden="true"
            className="user icon"
            onClick={[Function]}
          />
        </div>
      </div>
      <div
        className="field"
      >
        <label>
          Email
        </label>
        <div
          className="ui fluid left icon input"
        >
          <input
            name="email"
            onChange={[Function]}
            placeholder="Email"
            type="email"
            value=""
          />
          <i
            aria-hidden="true"
            className="envelope icon"
            onClick={[Function]}
          />
        </div>
      </div>
      <div
        className="field"
      >
        <label>
          Password
        </label>
        <div
          className="ui fluid left icon input"
        >
          <input
            name="password"
            onChange={[Function]}
            placeholder="Password"
            type="password"
            value=""
          />
          <i
            aria-hidden="true"
            className="lock icon"
            onClick={[Function]}
          />
        </div>
      </div>
      <button
        className="ui orange disabled button"
        disabled={true}
        onClick={[Function]}
        tabIndex={-1}
        type="submit"
      >
        <i
          aria-hidden="true"
          className="signup icon"
          onClick={[Function]}
        />
        Signup
      </button>
    </div>
  </form>,
  <div
    className="ui warning bottom attached message"
  >
    <i
      aria-hidden="true"
      className="help icon"
      onClick={[Function]}
    />
    Exiting user?
     
    <a
      href="/login"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Log in here
    </a>
    
    instead.
  </div>,
]
`;
